src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingComposter.java

```diff
@@ -77,7 +77,7 @@ public BuildingComposter(@NotNull final IColony c, final BlockPos l)
     {
         super(c, l);
 
-        keepX.put((stack) -> this.hasModule(ItemListModule.class) && this.getModuleMatching(ItemListModule.class, m -> m.getId().equals(COMPOSTABLE_LIST)).isItemInList(new ItemStorage(stack)), new Tuple<>(Integer.MAX_VALUE, true));
+        keepX.put((stack) -> this.getModuleMatching(ItemListModule.class, m -> m.getId().equals(COMPOSTABLE_LIST)).isItemInList(new ItemStorage(stack)), new Tuple<>(Integer.MAX_VALUE, true));
```

Comment:
Just removing the `hasModule` is not equivalent, unless you're certain it always will have one.  (On pain of null reference.)

Reply:
we are indeed certain that we have one, as this building is defined with one